spec.md

```diff
@@ -1616,6 +1625,14 @@ deprovision the service. This specification does not specify what a Service
 Broker is to do if it receives a deprovision request while there are still
 Service Bindings associated with it.
 
+If a Service Broker accepts the request to delete a Service Instance during
+the process of it being created, then it MUST have the net effect of halting
+the current creation process and deleting any resources associated with
+the Service Instance. If the request to create the Service Instance is
+asynchronous, then its `last_operation` response SHOULD return an HTTP
```

Comment:
If the broker is doing a bind asynchronously, and then it accepts an unbind asynchronously, I'd expect the `last_operation` endpoint to show the most recent operation, which is the unbind. So it seems wrong to have the `last_operation` then reflect the bind rather than the unbind. If there is an operation string to pass in, then the bind operation should definitely have that status, though.

Reply:
@kibbles-n-bytes I always assumed that the "operation" would be message-exchange specific - meaning the bind() and unbind() would have different "operation" values. The spec says: "A Service Broker-provided identifier for the operation."  And I *think* "operation" in that sentence means bind() vs unbind().  Or am I not following your question?